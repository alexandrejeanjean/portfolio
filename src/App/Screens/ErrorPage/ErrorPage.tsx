import React from 'react'
// import BubbleT from '../../Assets/BubbleT.svg'
import ClassicButton from '../../Shared/Button/Button'
import '../Home/home.scss'

const ErrorPage = () => {
  return (
    <>
      <main className='home-wrapper page'>
        <section>
          <div className='home-title-wrapper'>
            <div className='home-title-content'>
              <h1 className='home-title'>Erreur de calcul...</h1>
              <p className='home-subtitle'>
                Nouvelle itin√©raire juste en dessous
              </p>
              <ClassicButton text='Reprendre la route' endpoint='/' />
            </div>
          </div>
          <div className='home-bubble-wrapper'>
            {/* <img src={BubbleT} className='home-bubble' alt='' /> */}
          </div>
        </section>
      </main>
    </>
  )
}

export default ErrorPage
