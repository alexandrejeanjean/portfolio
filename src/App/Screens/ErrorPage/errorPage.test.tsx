import React from "react";
import { shallow } from "enzyme";
import ErrorPage from "./ErrorPage";

let wrapper: any;

beforeEach(() => {
  wrapper = shallow(<ErrorPage />);
});

describe("ErrorPage view", () => {
  it("should render ErrorPage component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
