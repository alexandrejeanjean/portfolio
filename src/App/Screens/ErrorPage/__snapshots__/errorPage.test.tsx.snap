// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage view should render ErrorPage component correctly 1`] = `
<Fragment>
  <main
    className="home-wrapper page"
  >
    <section>
      <div
        className="home-title-wrapper"
      >
        <div
          className="home-title-content"
        >
          <h1
            className="home-title"
          >
            Erreur de calcul...
          </h1>
          <p
            className="home-subtitle"
          >
            Nouvelle itin√©raire juste en dessous
            <Emoji
              onlyEmojiClassName="make-emojis-large"
              text="üëá"
            />
          </p>
          <ClassicButton
            endpoint="/"
            text="Reprendre la route"
          />
        </div>
      </div>
    </section>
  </main>
</Fragment>
`;
